collections/DBFilter.js

```diff
@@ -0,0 +1,27 @@
+Meteor.methods({
+  getFilter: function(date, days) {
```

Comment:
The description given in the tutorials on the meteor home page when making a call is:
1. The client sends a request to the server to run the method in a secure environment, just like an AJAX request would work
2. A simulation of the method runs directly on the client to attempt to predict the outcome of the server call using the available information
For me it seemed like a clean way to make the same filter available for both the server and the client.